client/src/app/employee/employee.component.ts

```diff
@@ -1,20 +1,41 @@
 import { Component } from '@angular/core';
-
-import { faLock, faUnlock } from '@fortawesome/free-solid-svg-icons';
+import { faPlus } from '@fortawesome/free-solid-svg-icons';
+import { Observable } from 'rxjs';
+import { Employee } from './employee-detail/state/employee.model';
+import { EmployeeQuery } from './employee-detail/state/employee.query';
+import { EmployeeService } from './employee-detail/state/employee.service';
+import { EmployeeListItem } from './employee-list/state/employee-list.model';
+import { EmployeeListQuery } from './employee-list/state/employee-list.query';
+import { EmployeeListService } from './employee-list/state/employee-list.service';
 
 @Component({
   selector: 'app-employee',
   templateUrl: './employee.component.html',
   styleUrls: ['./employee.component.scss'],
 })
 export class EmployeeComponent {
-  readonly faLock = faLock;
-  readonly faUnlock = faUnlock;
-  isReadonly = true;
+  readonly faPlus = faPlus;
+  employees: Observable<EmployeeListItem[]>;
+  employee: Observable<Employee>;
+  uiState = '';
```

Comment:
If you were to change this component to use `OnPush` change detection, you'd want to move this `uiState` into the `EmployeeStore` and react to changes to that state via the query.

This ends up being a pretty typically pattern - move state out of components into classes unrelated to rendering/DOM, then use the component as a binding between DOM and state.